line_profiler/_line_profiler.pyx

```diff
@@ -339,10 +339,23 @@ cdef class LineProfiler:
                 entries += list(cmap[entry].values())
             key = label(code)
 
-            entries = [(e["lineno"], e["nhits"], e["total_time"]) for e in entries]
+            temp_entries = [(e["lineno"], e["nhits"], e["total_time"]) for e in entries]
             # If there are multiple entries for a line, this will sort them by increasing # hits
-            entries.sort()
-
+            temp_entries.sort()
+            # If a line causes a branch, the stats for the True branch and False branch are separate.
+            # So we merge them here with 2 passes
+            branch_entries = {}
+            # Find the indicies of items with the same line number
+            for indx, entry in enumerate(temp_entries):
```

Comment:
This would be more readable as `for i, (lineno, nhits, total_time) in enumerate(temp_entries)`

Reply:
That's a good point, I'll fix that ASAP (see above comment for why ASAP might be a while).

Reply:
or `(for i, (lineno, _, _))` in this case